tests/nvme/rc

```diff
@@ -30,7 +30,9 @@ _nvme_requires() {
 		_have_modules nvmet nvme-core nvme-rdma nvmet-rdma
 		_have_configfs
 		_have_program rdma
-		_have_modules rdma_rxe || _have_modules siw
+		if [ -z "$use_hw_rdma" ]; then
```

Comment:
As described in "new" script, double brackets [[ ]] is recommended for blktests.

Reply:
I found that current blktests scripts already have many single brackets []. Still I prefer the double brackets, but I no longer stick to it. Feel free to leave the single brackets :)